basics/Main.java

```diff
@@ -0,0 +1,64 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Main {
+    public static void main(String[] args) {
+        //Test method
+        System.out.println("Main method is working!");
+
+        // pluralize
+        System.out.println(pluralize("dog", 2));
+        System.out.println(pluralize("cat", 1));
+
+        // flipping coins
+        System.out.println(flipNHeads(6));
+
+        // command line clock 
+        System.out.println(clock());
+
+    }
+
+    //pluralize method
+    public static String pluralize (String word, int number) {
+        String pluralWord = "";
```

Comment:
It looks like this won't work for the third given example input, "I own 0 turtles"; this would say "0 turtle" instead.